extensions/amp-script/0.1/amp-script.js

```diff
@@ -131,7 +125,45 @@ export class AmpScript extends AMP.BaseElement {
       onReceiveMessage: data => {
         dev().info(TAG, 'From worker:', data);
       },
-    }).then(workerDom => {
+      onMutationPump: this.mutationPump_.bind(this),
```

Comment:
Hmm. I'm pretty sure the naming of these has changed.

Reply:
Bad merge, removed.